### PR TITLE
Return sources when running Get and List

### LINKED ISSUE #14707: repo store: Return sources
In `internal/db/repos`:

Modify `Get`, `GetByName`, `GetByIDs`, `GetReposSetByIDs` and `List` to return repos with their associated sources

Extracted from 
https://github.com/sourcegraph/sourcegraph/issues/14092

### PR DESCRIPTION
**Pull request by asdine**
Only `Get` and `List` actually need to return the sources. The other `Get+` don't fetch the `RepoFields `columns.

Fixes #14707

### CODE REVIEW

File: cmd/frontend/types/types.go [hunk 1]
```diff
@@ -27,8 +27,30 @@ type RepoFields struct {
 	// Cloned is whether this repository is cloned.
 	Cloned bool
 
+	// Language is the primary programming language used in this repository.
```
Comment (ryanslade) [inline cmd/frontend/types/types.go:30]: This has been deprecated.

File: internal/db/repos.go [hunk 2]
```diff
@@ -179,7 +207,12 @@ var getBySQLColumns = []string{
 	"fork",
 	"archived",
 	"cloned",
+	"language",
```
Comment (ryanslade) [inline internal/db/repos.go:210]: This has been deprecated. It should be gone from `main`, maybe you haven't pulled recently?

**Comment by ryanslade** [comment]
I noticed that `getReposBySQL` uses `authzFilter`. We should confirm what this would mean if we use this function from `repo-updater`

**Review by ryanslade — APPROVED** [review]
LGTM once language is removed and confirmed the impact of `authzFilter`.

Also, it looks like `GetByName` is also fetching repos with sources, but that should be fine.

File: internal/db/repos.go (hunk 2 above)
Comment (asdine) [reply internal/db/repos.go]: Yes I forgot to pull before creating the new branch, I have removed it now

**Pull request merged**